### PR TITLE
First working solution

### PR DESCRIPTION
**Pull request by j0k3r**
This is the second init commit 🙂

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "20minutes"
```
Comment (ryuran) [inline .eslintrc]: maybe `"root": true`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    directories:
+        - node_modules
+
+env:
+    global:
+        - NODE_ENV=test
+        - TZ=Europe/Paris
+
+node_js:
+    - "8.10"
```
Comment (ryuran) [inline .travis.yml]: v10 is LTS now

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Use Case
+
+It validates GitHub Pull Requests against some specifications:
+- `body` should be at least 8 characters long
+- `title` should be at least 8 characters long
+
+## Prerequisites
+
+- Node.js 8.10
```
Comment (ryuran) [inline README.md]: v10 is LTS now

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+node_modules
```
Comment (mallier) [inline .gitignore]: It's not `/node_modules` or `node_modules/` for directory ?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+sudo: false
```
Comment (mallier) [inline .travis.yml]: `sudo: false` isn't the default behavior ?

File: .travis.yml (hunk 2 above)
Comment (mallier) [inline .travis.yml]: Can't we pass to node 10 ?

File: handler.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+  const body = JSON.parse(event.body)
+  // when creating the webhook
+  if (body && ('hook_id' in body)) {
+    response = {
+      statusCode: 200,
+      body: `Hello ${body.sender.login}, the webhook is now enabled for ${body.repository.full_name}, enjoy!`,
+    }
+
+    if (!body.hook.events.includes('pull_request')) {
```
Comment (mallier) [inline handler.js]: `body.hook.events` always exists ?

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+module.exports = {
+  entry: slsw.lib.entries,
+  stats: 'minimal',
+  target: 'node',
+  devtool: 'source-map',
+  externals: [nodeExternals()],
+  mode: slsw.lib.webpack.isLocal ? 'development' : 'production',
+  optimization: {
+    minimize: false,
```
Comment (mallier) [inline webpack.config.js]: Couldn't be ingore to use default value related to `mode` ?

File: .travis.yml (hunk 2 above)
Comment (j0k3r) [reply .travis.yml]: Yep but AWS Lambda only run 8.10 as max, not 10 yet.

Comment (j0k3r) [reply .travis.yml]: See https://github.com/20minutes/serverless-github-check/pull/1#discussion_r232693215

File: handler.js (hunk 6 above)
Comment (j0k3r) [reply handler.js]: Since I check for `hook_id` few lines above, yep

File: webpack.config.js (hunk 7 above)
Comment (j0k3r) [reply webpack.config.js]: Oh yeah!

File: handler.js (hunk 6 above)
Comment (mallier) [reply handler.js]: A little bit weird to doing that btw, what not checking `hook` existance rather that `hook_id` ?

File: .eslintrc (hunk 1 above)
Comment (j0k3r) [reply .eslintrc]: I wasn't aware of that configuration. Maybe we should add it to all our projects.

File: .gitignore (hunk 4 above)
Comment (tbrunel) [reply .gitignore]: if you don't specify a specific folder, this rule will ignore all node_modules folder in the whole project.

**Pull request merged**